app/models/user.rb

```diff
@@ -14,4 +14,16 @@ class User < ApplicationRecord
 
   has_many :orders
   belongs_to :merchant, optional: true
+
+  def admin?
```

Comment:
These methods become available automatically once you define an attribute as an enum so unless there's a particular reason these are defined here that I'm not catching, these three methods (`admin?`, `regular?` and `merchant_employee?`) can actually be deleted!